### PR TITLE
Managed COM sample for .NET Core 3.0

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
## Summary

Addition of a .NET COM server example

Requested in https://github.com/dotnet/standard/issues/412

cc @jeffschwMSFT @richlander @danharlin @jkoritzinsky @elinor-fung

### CODE REVIEW

**Comment by AaronRobinsonMSFT** [comment]
cc @rpetrusha @mairaw

File: core/extensions/COMServerDemo/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+COM Server Demo
+================
+
+This is a basic example of providing a managed COM server in .NET Core 3.0. Documentation on the inner workings of activation can be found at [here](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/COM-activation.md).
```
Comment (rpetrusha) [inline core/extensions/COMServerDemo/ReadMe.md]:
```suggestion
This is a basic example of providing a managed COM server in .NET Core 3.0. Documentation on the inner workings of activation can be found [here](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/COM-activation.md).
```

File: core/extensions/COMServerDemo/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Key Features
+------------
+
+Demonstrates how to consume provide a COM server in .NET Core 3.0 Preview 5 or later.
```
Comment (rpetrusha) [inline core/extensions/COMServerDemo/ReadMe.md]: how to consume, or how to provide?

Comment (AaronRobinsonMSFT) [reply core/extensions/COMServerDemo/ReadMe.md]: Oops.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @AaronRobinsonMSFT. I'll merge your PR now.

**Pull request merged**